code/modules/awaymissions/corpse.dm

```diff
@@ -33,6 +33,7 @@
 	var/banType = ROLE_GHOST
 	var/ghost_usable = TRUE
 	var/offstation_role = TRUE // If set to true, the role of the user's mind will be set to offstation
+	var/death_cooldown = 0 // How long you have to wait after dying before using it again, in minutes. People that join as observers are not included.
```

Comment:
I'd personally prefer this is was in deciseconds so a coder can directly use the time macros instead of just say `5`.